fission-cli/library/Fission/CLI/Parser/Command/Setup.hs

```diff
@@ -62,6 +72,14 @@ email = do
     ""   -> Nothing
     mail -> Just mail
 
+keyfile :: ReadM (Maybe FilePath)
+keyfile = do
+  raw <- str
+  pure case raw of
+    ""   -> Nothing
+    path -> Just path
```

Comment:
Nothing to change in this PR, but I wonder if we should start collecting these in their own module to share between parsers 🤔